src/test/java/com/google/javascript/clutz/refer_from_default_ns.js

```diff
@@ -0,0 +1,10 @@
+goog.provide('fn');
+goog.provide('fn.String');
+
+/** @return {fn.String} */
```

Comment:
not sure how this test is broken without your changes. Namespace should be empty to trigger the bug, but in the test it looks like 'fn'?